Starting with GHC 9.4.4 this test is flaky for some reason, i.e.
sometimes it passes and sometimes it fails on an identical build.

diff -upr ghc-9.4.4.orig/testsuite/tests/driver/all.T ghc-9.4.4/testsuite/tests/driver/all.T
--- ghc-9.4.4.orig/testsuite/tests/driver/all.T	2023-02-04 09:26:43.401678280 +0100
+++ ghc-9.4.4/testsuite/tests/driver/all.T	2023-02-04 09:27:15.881724610 +0100
@@ -170,7 +170,6 @@ test( 'T4114d', [fobject_code, expect_br
 
 test('T5584', [], makefile_test, [])
 test('T5198', [], makefile_test, [])
-test('T7060', [], makefile_test, [])
 # N.B. The typo in the argument below is intentional.
 test('T7130', normal, compile_fail, ['-fflul-laziness'])
 test('T7563', when(unregisterised(), skip), makefile_test, [])
